indispensable/build.gradle.kts

```diff
@@ -182,6 +182,7 @@ kotlin {
         commonTest {
             dependencies {
                 implementation(kotest("property"))
+                api("io.arrow-kt:arrow-core:1.2.4")
```

Comment:
why is this an api dependency now?

Reply:
good catch! this is a leftover from some trial-and error, regarding https://github.com/arrow-kt/arrow/issues/3490. it was pulled in anyway by KmmResult. gone now!